src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor({ health = 100, name }) {
+    this.health = health;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
```

Comment:
The constructor of the `Herbivore` class should directly take `name` as a parameter and pass it to the `super` call as `super({ name })` to match the expected structure in the `Animal` class constructor.